src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -2714,7 +2714,7 @@ class C
 
     static void M()
     {
-        IComparable<long> c = Goo<long>(1, 1);
+        IComparable<long> c = Goo(1, (long)1);
```

Comment:
@DustinCampbell see https://github.com/dotnet/roslyn/pull/26237/files#r182592374

This is the same as the above test.  Humorously, this test is called "InsertCastToKeepGenericMethodInference".  So it sounds like we did actually *intend* for their to be a cast, rather than keeping the type parameters.